src/os/filestore/FileJournal.cc

```diff
@@ -189,39 +189,37 @@ int FileJournal::_open_file(int64_t oldsize, blksize_t blksize,
   }
 
   if (create && (oldsize < conf_journal_sz)) {
-    uint64_t newsize(cct->_conf->osd_journal_size);
-    newsize <<= 20;
-    dout(10) << "_open extending to " << newsize << " bytes" << dendl;
-    ret = ::ftruncate(fd, newsize);
+    dout(10) << "_open extending to " << conf_journal_sz << " bytes" << dendl;
```

Comment:
also i'd suggest continue using `new_size` but initialize it using
```c++
const uint64_t newsize(conf_journal_sz);
```
as it helps with the readability, imo.